src/core/constants.js

```diff
@@ -97,6 +97,13 @@ export const VIEW_CONTEXT_HOME = 'VIEW_CONTEXT_HOME';
 export const SEARCH_SORT_POPULAR = 'hotness';
 export const SEARCH_SORT_TOP_RATED = 'rating';
 
+// Operating system for add-ons
+export const OS_WINDOWS = 'windows';
+export const OS_MAC = 'mac';
+export const OS_LINUX = 'linux';
+// TODO: Do we need this? We have clientApp.
```

Comment:
Yeah, I expected to see it as an option in the operating system drop down. Users will need to filter for Android only extensions somehow. Is that happening in a separate patch?

Reply:
It might be worth doing as a separate patch because changing this OS flag would change the `clientApp` in the URL, not the OS in the search params.

I'd rather do it in a follow-up patch as it's different behaviour and sort of unrelated to this change. If anything it should be a select/switch/etc. elsewhere that is site-wide.